src/main/java/com/google/idea/perf/tracer/ui/TracerPanel.kt

```diff
@@ -239,6 +243,7 @@ class TracerPanel(
         val stats = CallTreeUtil.computeFlatTracepointStats(treeSnapshot)
         listView.setTracepointStats(stats)
         treeView.setCallTree(treeSnapshot)
+        configView.setTracingConfig(TracerUserConfig.cloneUserTraceRequests())
```

Comment:
Should this call be moved into `TracerController` (after new tracing commands are issued), rather than here inside the render loop?

Reply:
`configView` seems similar to previous set calls in the same function. Controller wouldn't have access to it. Could you elaborate please if you think it's worth moving a call?